src/web/mjs/connectors/Hiperdex.mjs

```diff
@@ -7,6 +7,29 @@ export default class Hiperdex extends WordPressMadara {
         super.id = 'hiperdex';
         super.label = 'Hiperdex';
         this.tags = [ 'hentai', 'webtoon', 'english' ];
-        this.url = 'https://hiperdex.com';
+        this.config = {
```

Comment:
Seems to me once pushed to the user, the URL can no longer be automatically be updated by the developers, the user must always manually update the URL.
Is this by intention?

Reply:
Its like this for connectors where url is in settings afaik. Like i've said i would have detected the url  in initialize instead of putting a setting but Hiperdex also sometimes fire Cloudflare and one of the two domains may be unreachable.

I can ofc change to usual domain detection but there is no "master domain".

Reply:
Maybe use a `'select'` [type setting](https://github.com/manga-download/hakuneko/blob/20c5e61c46b71de379a4635a21018f406d32c42b/src/web/mjs/engine/Settings.mjs#L135-L141) with pre-defined mirrors instead of a `'text'` type to make it more suitable for users?